### ISSUE TITLE
Some buttons don't have accessible names.

### ISSUE THREAD

**Issue by Roger-Heathcote** [jackherizsmith/jackherizsmith.github.io#1]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers. Buttons should either contain some text e.g. <button>a name</button>

Or it should use the "aria-label" attribute...

e.g. <button aria-label="a name"><button>

Failing Elements
--
button#carouselButton0.carousel-button
button#carouselButton1.carousel-button
button#carouselButton2.carousel-button
button#carouselButton3.carousel-button
button#carouselButton4.carousel-button
button#carouselButton5.carousel-button
